changelogs/unreleased/inherit-strict-mode-for-tap-subtests.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/lua
+
+ * Tap subtests inherit strict mode from parent.
```

Comment:
1. Nit: please, get rid of the leading whitespace.
2. I would add the pull request id, since there is no issue.

Reply:
Fixed. Thanks.